lib/phoenix/endpoint/supervisor.ex

```diff
@@ -389,10 +389,11 @@ defmodule Phoenix.Endpoint.Supervisor do
 
   defp warmup_static(endpoint, %{"latest" => latest, "digests" => digests}) do
     Phoenix.Config.put_new(endpoint, :cache_static_manifest_latest, latest)
+    with_vsn? = endpoint.config(:url)[:vsn]
```

Comment:
:vsn is not really a property of url. We would need to create a specific entry for it instead of putting it inside url. :)